src/options.go

```diff
@@ -110,6 +113,11 @@ const usage = `usage: fzf [options]
     FZF_DEFAULT_OPTS      Default options
                           (e.g. '--layout=reverse --inline-info')
 
+  Development
+    --cpuprofile=FILE      Write a CPU profile to the specified file.
+    --memprofile=FILE      Write an allocation profile to the file before exiting.
+    --blockprofile=FILE    Write a goroutine blocking profile to the specified file.
+    --mutexprofile=FILE    Write a mutex contention profile to the specified file.
```

Comment:
I'd like to not show these options in `--help` option. Because ordinary users don't ever need to know them. Can you add these to the man page instead?

Reply:
Done